pkg/projects/Microsoft.NETCore.Console/project.json

```diff
@@ -0,0 +1,51 @@
+{
+  "dependencies": {
+    "Microsoft.NETCore" :"5.0.1-rc3-24123-01",
```

Comment:
These versions are now out of date, as I updated to 24126 this morning. If you want to update them you can run:

`powershell build_projects\update-dependencies\update-dependencies.ps1 -Targets UpdateFiles` from the root of the repo.